### DISCUSSION TITLE
Servo motors become unresponsive after code from this library executes.

### DISCUSSION THREAD

**Discussion by ManavP16** [ArminJo/Talkie#33] [General]
I'm trying to use this library to add some speech for a project. 
I have some servo motors, LEDs and an 8ohm 0.25W speaker. 
Initially, everything works, however, once the code executes a line for the speaker (using this library), the servo motors stop working, but the LEDs continue to work as coded. When I reset the board, the servo motors work and this process begins again.
Is there some line of code... or anything else I can use to help solve my issue? Thanks!

**Comment by ArminJo** [comment]
**Timer 1** (Servo timer) is used at all ATmegas for updating voice output data at 8 kHz.<br/>
**Timer 2** (Tone timer) on ATmega328 (62500 Hz / 16 µs) or **Timer 4** on ATmega2560 + ATmega32U4 (5 µs) is used to generate the 8 bit PWM output.<br/>
Both timers are therefore not available for other libraries / purposes, like servo control or tone output.

**Reply by ManavP16** [reply]
Hey ArminJo,
Thanks for the reply!
What I got from your response is that I cannot control the servos while using the library.
Are there any other alternatives I can use to achieve similar results that you are aware of?

**Reply by ArminJo** [reply]
Try PCA9685 as Servo expander.

**Reply by ManavP16** [reply]
I am not proficient at Arduino yet, is there a tutorial I can follow to implement that change?
